### ISSUE TITLE
Needed basic design for Signup, Login and Password Reset page

### ISSUE THREAD

**Issue by Dinesh100K** [ManVTish/rails_cart#8]
<img src="https://user-images.githubusercontent.com/31548894/181905993-27e76b6e-e3c3-48a3-9a3b-5b953d2b233f.png" width="600">

<img src="https://user-images.githubusercontent.com/31548894/181906110-f1b6ffbd-4f28-49cb-b500-f729a1e680e2.png" width="600">

<img src="https://user-images.githubusercontent.com/31548894/181906139-b628f186-2c97-4d70-bc38-96c2e0efc242.png" width="600">

**Comment by Dinesh100K** [comment]
@ManVTish What do you think? Do we need to add design first or first complete our backend part.

**Comment by ManVTish** [comment]
@Dinesh100K I agree with you, we should setup a basic signup/signin with proper authorization first. That can be counted as first feature to this web app.

**Comment by ManVTish** [comment]
Currently we have signup page, signin & reset page needs work.
